tests/tck/features/optimizer/PushSampleDownRule.feature

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Push Limit down rule
+
+  Background:
+    Given a graph with space named "nba"
+
+  Scenario: push limit down to GetNeighbors
+    When profiling query:
+      """
+      GO 1 STEPS FROM "James Harden" OVER like REVERSELY LIMIT [2]
+      """
+    Then the result should be, in any order:
+      | like._dst         |
+      | "Dejounte Murray" |
+      | "Luka Doncic"     |
+    And the execution plan should be:
+      | id | name         | dependencies | operator info  |
+      | 5  | Project      | 6            |                |
+      | 6  | Limit        | 7            |                |
```

Comment:
Can the limit node be deleted(bcz it's being pushed down)？

Reply:
Push down only perform in single storage node, when graph collect the many results from different storage we need limit again.

Reply:
ACK.